tools/regtesting/timings.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# author: Ole Schuett
+
+import sys
+import math
+from __future__ import print_function
+
+#===============================================================================
+def main():
+    if(len(sys.argv) != 2):
+        print("Usage: timings.py <timings.txt>")
+        sys.exit(1)
+
+    fn = sys.argv[1]
+    timings = sorted([float(line.split()[0]) for line in open(fn).readlines()])
```

Comment:
please use

```python
with open(fn) as fhandle:
    timings = sorted(float(line.split()[0]) for line in fhandle.readlines())
```

... because we all know that people copy stuff around and suddenly you have a fd leak in a loop.
Furthermore: `sorted()` can handle generator expressions.

Reply:
I didn't know about sorted(). That's indeed nice.